manager/integration/tests/test_migration.py

```diff
@@ -147,7 +146,55 @@ def test_migration_with_failed_replica(clients, volume_name):  # NOQA
        And the old failed replica will be cleaned up.
     8. Validate initially written test data.
     """
-    return
+    def finalizer():
+        exec_cmd = ["mkdir", "/var/lib/longhorn/replicas"]
+        subprocess.check_output(exec_cmd)
+
+    request.addfinalizer(finalizer)
+
+    client, volume, data = setup_migration_test(clients,
+                                                volume_name,
+                                                replica_cnt=3)
+
+    current_node = common.get_self_host_id()
+    hosts = get_hosts_for_migration_test(clients)
+    migrate_target = hosts[0]
+    volume.attach(hostId=current_node)
+    volume = common.wait_for_volume_healthy(client, volume_name)
+
+    old_replicas = []
+    for replica in volume.replicas:
+        old_replicas.append(replica.name)
+
+    exec_cmd = ["rm", "-rf",  "/var/lib/longhorn/replicas"]
```

Comment:
Probably Yes. The replica directory name is already exposed to HTTP API

Reply:
Hi @innobead ,
In my environment this command will keep` /var/lib/longhorn `exist, could you guide more detail for this comment? thank you.

Reply:
We can crash the replica/make the volume degraded by removing the specific replica directory only. Removing the whole data directory `/var/lib/longhorn` may affect all replicas in this disk.

Reply:
Exactly. @cchien816 Because right now our e2e testing does not run in parallel, so it should be fine, but suggest updating the case via deleting the specific replica folder in the future.